### DISCUSSION TITLE
Removing visualization of bounding box and label on keypoints inference

### DISCUSSION THREAD

**Discussion by asluborski** [facebookresearch/detectron2#3279] [Q&A] [answered]
I was wondering if there is a flag/command or where in the code I could disable the visualization of bounding box and label and just display the keypoint joints.

**Comment by mavisw** [comment, marked as answer]
Hi, 
Not sure if you've got the answer, but I was looking for the same solution and found it in another thread below.
Tried, and it works. You can refer to the 'visualizer.py'
Hope it helps!

https://github.com/facebookresearch/detectron2/discussions/2992
